CHANGELOG.md

```diff
@@ -0,0 +1,331 @@
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.8.1+
+
+# 0.8.0
+## 06/02/2016
+
+* The main new feature in this release is the `register` subcommand which 
+can be used to register an account with the Let's Encrypt CA. 
+* Additionally, you can run `certbot register --update-registration` to 
+change the e-mail address associated with your registration. 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.8.0+
+
+# 0.7.0
+## 05/27/2016
+
+* `--must-staple` to request certificates from Let's Encrypt with the 
+OCSP must staple extension 
+* automatic configuration of OSCP stapling for Apache 
+* requesting certificates for domains found in the common name of a 
+custom CSR 
+* a number of bug fixes 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=milestone%3A0.7.0+is%3Aissue 
+
+# 0.6.0
+## 05/12/2016 
+
+* Renamed the client from `letsencrypt` to `certbot`
+* Fixed a small json deserialization error
+* Versioned the datetime dependency in setup.py
+* Preserve domain order in generated CSRs
+* Some minor bug fixes
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?utf8=%E2%9C%93&q=is%3Aissue%20milestone%3A0.6.0%20is%3Aclosed%20
+
+# 0.5.0
+## 04/05/2016
+
+* The ability to use the webroot plugin interactively. 
+* The flags --pre-hook, --post-hook, and --renew-hook which can be used 
+with the renew subcommand to register shell commands to run in 
+response to renewal events. Pre-hook commands will be run before 
+any certs are renewed, post-hook commands will be run after any 
+certs are renewed, and renew-hook commands will be run after each 
+cert is renewed. If no certs are due for renewal, no command is run. 
+* Cleaner renewal configuration files. In /etc/letsencrypt/renewal by 
+default, these files can be used to control what parameters are used 
+when renewing a specific certificate. 
+* A -q/--quiet flag which silences all output except errors. 
+* An --allow-subset-of-domains flag which can be used with the renew 
+command to prevent renewal failures for a subset of the requested 
+domains from causing the client to exit. 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/letsencrypt/letsencrypt/issues?q=milestone%3A0.5.0+is%3Aissue 
+
+# 0.4.2
+## 03/03/2016
+
+* Resolves problems encountered when compiling letsencrypt 
+against the new OpenSSL release. 
+* A patch fixing problems of using letsencrypt renew with configuration files
+from private beta has been added. 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/letsencrypt/letsencrypt/issues?q=is%3Aissue+milestone%3A0.4.2 
+
+# 0.4.1
+## 02/29/2016
+
+* Apache parsing errors with some configurations 
```

Comment:
This list doesn't really make sense out of context. In the original e-mail, I had the following text above these bullet points:

> This is a bug fix release which includes patches for the following
problems:

Listing those problems on their own is a bit awkward. I'd prepend a word like "fixes", "resolves", or whatever other word your favorite thesaurus comes up with so these phrases make more sense on their own.